packages/plugin-ext/src/main/node/paths/plugin-paths-service.ts

```diff
@@ -139,15 +140,9 @@ export class PluginPathsServiceImpl implements PluginPathsService {
     }
 
     private async cleanupOldLogs(parentLogsDir: string): Promise<void> {
-        // @ts-ignore - fs-extra types (Even latest version) is not updated with the `withFileTypes` option.
-        const dirEntries = await readdir(parentLogsDir, { withFileTypes: true }) as string[];
-        // `Dirent` type is defined in @types/node since 10.10.0
-        // However, upgrading the @types/node in theia to 10.11 (as defined in engine field)
-        // Causes other packages to break in compilation, so we are using the infamous `any` type...
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        const subDirEntries = dirEntries.filter((dirent: any) => dirent.isDirectory());
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        const subDirNames = subDirEntries.map((dirent: any) => dirent.name);
+        const dirEntries = await readdir(parentLogsDir, { withFileTypes: true }) as fs.Dirent[];
+        const subDirEntries = dirEntries.filter((dirent: fs.Dirent) => dirent.isDirectory());
+        const subDirNames = subDirEntries.map((dirent: fs.Dirent) => dirent.name);
```

Comment:
Minor: I don't think we need the type casting here.

```suggestion
        const dirEntries = await readdir(parentLogsDir, { withFileTypes: true });
        const subDirEntries = dirEntries.filter(dirent => dirent.isDirectory());
        const subDirNames = subDirEntries.map(dirent => dirent.name);
```